pkg/build/util/consts.go

```diff
@@ -112,4 +112,8 @@ const (
 	// CustomBuildStrategyBaseImageKey is the environment variable that indicates the base image to be used when
 	// performing a custom build, if needed.
 	CustomBuildStrategyBaseImageKey = "OPENSHIFT_CUSTOM_BUILD_BASE_IMAGE"
+
+	// AdditionalTrustedCAKey is the ConfigMap key for the certificate bundle containing additional
+	// trusted CAs used by builds.
+	AdditionalTrustedCAKey = "additional-ca.crt"
```

Comment:
per my comment on your builder PR, let's align these names so we're not changing the filename when copying it.

Reply:
`additional-ca.crt` is the file name, per openshift/builder#21